controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -101,6 +101,11 @@ public ResponseEntity<Object> fetchPdf(String claimSubmissionId)
       FetchPdfResponse pdfResponse = objectMapper.readValue(response, FetchPdfResponse.class);
       log.info("RESPONSE from fetchPdf returned status: {}", pdfResponse.getStatus());
       if (pdfResponse.hasContent()) {
```

Comment:
NOT_FOUND and ERROR still returns 200 because pdfResponse.hasContent is false and your logic never comes into play.